### DISCUSSION TITLE
Lab4-2 mprj問題

### DISCUSSION THREAD

**Discussion by Alex17898** [bol-edu/HLS-SOC-Discussions#123] [Q&A] [answered]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/134195336/b73839c7-e005-4187-8cf1-04d5705f3b4e)
想請問一下講義這裡所說的mprj[23:16]是指mprj_io這跟pin嗎?
如果要寫入這根pin的話，是否是透過User project中的io_out來寫入，又或者是透過其他方式寫入?
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/134195336/57222744-5ee2-491e-a057-7ebaaff4439f)

**Comment by LingChiYang** [comment, marked as answer]
同學好

由於先前program IO至mgmt core (可參照main function)，mprj_io會由mgmt core控制，因此需要經由mgmt寫入mprj_io。
如果要寫入這些pin，需要在main function修改program io。
